enterprise/management/commands/assign_enterprise_user_roles.py

```diff
@@ -0,0 +1,35 @@
+"""
+management command for assigning existing enterprise users to their respective enterprise roles.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+from enterprise.constants import ENTERPRISE_ADMIN_ROLE, ENTERPRISE_DATA_API_ACCESS_GROUP, ENTERPRISE_LEARNER_ROLE
+from enterprise.models import EnterpriseCustomerUser, SystemWideEnterpriseRole, SystemWideEnterpriseUserRoleAssignment
+
+
+class Command(BaseCommand):
+    """ management command class """
+    help = 'Assigns existing enterprise users to their specific roles.'
+
+    def handle(self, *args, **options):
+        enterprise_admin_role = SystemWideEnterpriseRole.objects.get(name=ENTERPRISE_ADMIN_ROLE)
+        enterprise_learner_role = SystemWideEnterpriseRole.objects.get(name=ENTERPRISE_LEARNER_ROLE)
+
+        # Assign admin role to users with enterprise data api access.
+        users_with_enterprise_access = User.objects.filter(groups__name=ENTERPRISE_DATA_API_ACCESS_GROUP)
+        for user_with_enterprise_access in users_with_enterprise_access:
+            SystemWideEnterpriseUserRoleAssignment.objects.update_or_create(
+                user=user_with_enterprise_access,
+                role=enterprise_admin_role
+            )
+
+        # Assign enterprise learner role to enterprise customer users.
+        enterprise_learners = EnterpriseCustomerUser.objects.all()
```

Comment:
we might want to separate this from the logic above as there are about 55 thousand of these records compared to those in the group (not more than 100). because of the size, we also should structure the command for enterprise learners so that we are batching these updates. see https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/management/commands/compute_grades.py#L57